core/admin/mailu/configuration.py

```diff
@@ -68,6 +68,8 @@
     'LOGO_URL': None,
     'LOGO_BACKGROUND': None,
     # Advanced settings
+    'PROXY_CREATE': False,
+    'PROXY_SECRET': None,
```

Comment:
We need to ensure that PROXY_SECRET is not brute-forceable (check that it's at least 16 chars or something)